core/src/utils/test/playwright/playwright-declarations.ts

```diff
@@ -4,7 +4,13 @@ import type { TestConfig } from './generator';
 import type { EventSpy } from './page/event-spy';
 import type { LocatorOptions, E2ELocator } from './page/utils/locator';
 
-export interface E2EPageOptions extends PageOptions, TestConfig {}
+export interface E2EPageOptions extends PageOptions, TestConfig {
+  /**
+   * Indicates if the page content is set inline with `.setContent`. This setting can
+   * only be true or not set.
+   */
+  inline?: true;
```

Comment:
The use of `true` here is intentional. This configuration should only be enabled or be not defined.